frontend/public/components/search-filter-dropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem, TextInput } from '@patternfly/react-core';
+import { CaretDownIcon, FilterIcon } from '@patternfly/react-icons';
+
+export const SearchFilterDropdown: React.SFC<SearchFilterDropdownProps> = (props) => {
+  const [isOpen, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState('Label');
```

Comment:
Ideally we'd have an enum for label vs name instead of strings. If we use strings, use `label` (lowercase)

Reply:
Like the 'All' issue, I use the name programatically for the toolbar and I would have to capitalize the values otherwise.  For simplicity, I would like to leave this capitalized.